doc/centos-tutorial.txt

```diff
@@ -0,0 +1,141 @@
+This document outlines the necessary steps to take to install and compile MANA
+on a new CentOS machine, as well as launch, checkpoint and restart a given MPI
+application.
+
+Root privileges are required to install many of the necessary packages, and this
+tutorial will assume you have them.
+
+A. INSTALLING AND COMPILING MANA
+
+1. Installing essential packages - this section can be skipped depending on the
+existing packages on your machine.
+
+  # yum install -y git
+  # yum groupinstall 'Development Tools'
+  # yum install -y mlocate vim wget
+  # yum install -y centos-release-scl
+  # yum install -y devtoolset-8-gcc devtoolset-8-gcc-c++
+  # echo 'source scl_source enable devtoolset-8' > ~/.bashrc
+  # source ~/.bashrc
```

Comment:
This next section is about using an available MPI, such as MPICH.
Let's add a marker for this section like:
### INSTALLING MPI ###

Also, let's itemize what the MPI package provides, so that we can make MANA more independent of a particular MPI implementation such as MPICH.  Probably, we need 'mpicc' and 'mpic++' and 'libmpi.so' and 'libmpi.a' and 'mpi.h'.  What are the locations assumed by MANA?  Maybe mpicc and mpic++ in the user's path, and libmpi.so/libmp.a/mpi.h in the standard lib and include locations.  If the user has installed libmpi.so/libmp.a/mpi.h, what should the user change in MANA (or change by setting environment variables for LD_LIBRARY_PATH, etc.)?  (For example, it would be nice to make this compatible with a new experimental MPI).
    For this first draft, this information doesn't have to be complete, but let's point to some common choices here.

Reply:
is there a resource that provides a list of the libraries necessary for MANA?

Reply:
As we discussed, it's enough if we list the libraries that we are aware of.  No need to spend a lot of time with a deep analysis to list _all_ libraries.  We can extend the tutorial later, based on comments from newer people who try out the tutorial.